wiki/Tournaments/OSIM/1/en.md

```diff
@@ -0,0 +1,597 @@
+---
+tags:
+  - OSIM
+---
+
+# osu! Singapore-Indonesia-Malaysia Tournament
+
+![OSIM logo](img/logo.jpg)
+
+The **osu! Singapore-Indonesia-Malaysia Tournament** (***OSIM***) was a double-elimination 2v2 osu!standard tournament hosted by  ![][flag_SG] [youi](https://osu.ppy.sh/users/7537133), ![][flag_ID] [Fre](https://osu.ppy.sh/users/3484548), and ![][flag_MY] [Kumaxione](https://osu.ppy.sh/users/2582531). The tournament was open to all players from Singapore, Indonesia, and Malaysia regardless of rank. It was the 1st iteration of the tournament series.
```

Comment:
```suggestion
The **osu! Singapore-Indonesia-Malaysia Tournament** (***OSIM***) was a double-elimination 2v2 osu!standard tournament hosted by ![][flag_SG] [youi](https://osu.ppy.sh/users/7537133), ![][flag_ID] [Fre](https://osu.ppy.sh/users/3484548), and ![][flag_MY] [Kumaxione](https://osu.ppy.sh/users/2582531). The tournament was open to all players from Singapore, Indonesia, and Malaysia regardless of rank. It was the first iteration of the tournament series.
```
You can also replace tournament series with osu! Singapore Indonesia-Malaysia Tournament series